pkg/option/config.go

```diff
@@ -3058,6 +3082,19 @@ func (c *DaemonConfig) checkIPv4NativeRoutingCIDR() error {
 	return nil
 }
 
+// check the tunnel endpoint IPv4 cidr can match an actual interface IP
+func (c *DaemonConfig) checkIPv4TunnelEndpointCIDR() error {
+	tunnelEndpointIPv4CIDR := c.GetTunnelEndpointIPv4CIDR()
+	log.WithField(logfields.CIDR, tunnelEndpointIPv4CIDR).Info("Check Tunnel Endpoint IPv4 CIDR")
```

Comment:
:+1: better use of `log.WithField(..)` / `log.WithFields(...)` here but again this looks more like debug logging rather than Info that all users may need.

Reply:
Thanks!